src/UserService.Models.Dto/Requests/Password/ReconstructPasswordRequest.cs

```diff
@@ -4,12 +4,13 @@
 namespace LT.DigitalOffice.UserService.Models.Dto
 {
   public record ReconstructPasswordRequest
+  //password and secred must be receiven from body! receiving from query is not safe
```

Comment:
is not secure maybe?

Reply:
'secure' was the first version XD
OK, 'secure' is back